proto/beacon/p2p/v1/types.proto

```diff
@@ -183,7 +183,7 @@ message DepositParameters {
 message ProposalSignedData {
   uint64 slot = 1;
   uint64 shard = 2;
-  bytes block_hash_hash32 = 3;
+  bytes block_hash32 = 3;
```

Comment:
Might as well change this to `root` and all of the `hash` references to `root` as well